snap/snapcraft.yaml

```diff
@@ -90,16 +89,17 @@ parts:
     - -*
     build-packages:
     - g++
-    - ldc
     - libedit-dev
     - ninja-build
     - zlib1g-dev
+    build-snaps:
+    - ldc2/1.11/stable
```

Comment:
Just as a note, I wonder if I shouldn't insist on the oldest LDC version that will work (but it's probably not a big deal).